.semaphoreci/setup.sh

```diff
@@ -1,6 +1,7 @@
 # For personnal CI
 # mv /home/runner/workspace/src/github.com/<username>/ /home/runner/workspace/src/github.com/containous/
 # cd /home/runner/workspace/src/github.com/containous/maesh/
+for s in apache2 cassandra elasticsearch memcached mysql mongod postgresql sphinxsearch rethinkdb rabbitmq-server redis-server; do sudo service $s stop; done
```

Comment:
We are using the docker light box, where these are not present no?

Reply:
We are no more using docker light because there are too many flaky e2e tests with this type of boxes.